testing/trino-tests/src/test/java/io/trino/execution/TestEventListenerBasic.java

```diff
@@ -413,13 +413,13 @@ public void testReferencedTablesWithMaterializedViews()
     public void testReferencedTablesInCreateView()
             throws Exception
     {
-        runQueryAndWaitForEvents("CREATE VIEW mock.default.test_view AS SELECT * FROM nation", 2);
+        runQueryAndWaitForEvents("CREATE VIEW mock.default.create_view_not_exist AS SELECT * FROM nation", 2);
```

Comment:
`create_view_not_exist` -> `another_test_view` or something similar

Reply:
the view name need start with ``create`` because of withGetTableHandle in mock,so can we use ``create_another_test_view``?